File: 프로그래머스/lv1/12926. 시저 암호/시저 암호.cs
public class Solution
{
    public string solution(string s, int n)
    {
        string big = "ABCDEFGHIJKLMNOPQRSTUVWXYZ";
        string small = "abcdefghijklmnopqrstuvwxyz";

        string answer = "";

        foreach (char l in s)
        {
            string now = l.ToString();

            if (big.Contains(now))
            {
                answer += big[(big.IndexOf(now) + n) % 26];
            }
            else if (small.Contains(now))
            {
                answer += small[(small.IndexOf(now) + n) % 26];
            }
            else
                answer += " ";
        }

        return answer;
    }
}

File: 프로그래머스/lv1/12926. 시저 암호/시저 암호.py
def solution(s, n):
    small = 'abcdefghijklmnopqrstuvwxyz'
    big = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ'
    
    return ''.join(small[(ord(l) - 97 + n)%26] if l.islower() else (big[(ord(l) - 65 + n)%26] if l.isupper() else ' ') for l in s)